### DISCUSSION TITLE
Issue with RelationManager & Multi-Tenancy

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#8502] [General]
Hello I have the exact same problem as here

- Error is : `Field 'team_id' doesn't have a default value` when attaching a model to another (HABTM)
- Issue 7511 : https://github.com/filamentphp/filament/issues/7511

but the issue is marked closed.

<img width="1140" alt="Screenshot 2023-09-14 at 20 42 47" src="https://github.com/filamentphp/filament/assets/5849907/37a51bfa-33bc-4acd-9715-07e0b9db52da">
<img width="716" alt="Screenshot 2023-09-14 at 20 42 41" src="https://github.com/filamentphp/filament/assets/5849907/ec9f59a5-2823-4fa6-b0cb-4e6d0224de1f">

**Comment by ghost** [comment]
@danharrin Any idea? Thanks!

**Comment by danharrin** [comment]
```php
->mutateFormDataUsing(function ($data) {
    $data['team_id'] = Filament::getTenant()->id;

    return $data;
})
```

make sure `team_id` is in the relationship pivot
